File: examples/python/src/generate_event.py
import time
from platform_os import is_windows

if is_windows():
    from request import post
else:
    from sock import post

json_response = post(
    "/events",
    {
        "key": "test-key",
        "detail": "test-detail",
        "occurred_at": int(time.time() * 1000),
    },
)

print("The response is as follows.\n")
print(json_response)

File: examples/nodejs/src/generateEvent.js
import { post } from "./lib/sock.js";

(async () => {
  const payload = {
    key: "test-key",
    detail: "test-detail",
    occurred_at: Date.now(),
  };
  const response = await post("/events", payload);

  console.log("The response is as follows.\n");
  console.log(response);
})();
